charts/templates/configmap.yaml

```diff
@@ -0,0 +1,37 @@
+{{- $namespace := .Values.common.namespace | required "A valid .common.namespace is required!" -}}
+{{- $yunikornHost := .Values.yunikorn.host | required "A valid .yunikorn.host is required!" -}}
+{{- $yunikornPort := .Values.yunikorn.port | required "A valid .yunikorn.port is required!" -}}
+{{- $yunikornProtocol := .Values.yunikorn.protocol | required "A valid .yunikorn.protocol is required!" -}}
+{{- $dbHost := .Values.db.host | required "A valid .db.host is required!" -}}
+{{- $dbPort := .Values.db.port | required "A valid .db.port is required!" -}}
+{{- $dbName := .Values.db.name | required "A valid .db.name is required!" -}}
+{{- $dbUser := .Values.db.user | required "A valid .db.user is required!" -}}
+{{- $dbPoolMaxConns := .Values.db.poolMaxConns | required "A valid .db.poolMaxConns is required!" -}}
+{{- $dbPoolMinConns := .Values.db.poolMinConns | required "A valid .db.poolMinConns is required!" -}}
+{{- $dbPoolMaxConnLifetime := .Values.db.poolMaxConnLifetime | required "A valid .db.poolMaxConnLifetime is required!" -}}
+{{- $dbPoolMaxConnIdleTime := .Values.db.poolMaxConnIdleTime | required "A valid .db.poolMaxConnIdleTime is required!" -}}
+{{- $yhsServerAddr := .Values.yhsServerAddr | required "A valid .yhsServerAddr is required!" -}}
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: yhs-config
+  namespace: {{ $namespace }}
```

Comment:
Omit namespace everywhere, it will be set with the `--namespace` flag during `helm install`